fccodegen/tests/shared.py

```diff
@@ -0,0 +1,181 @@
+#
+# Shared classes and methods for testing.
+#
+import io
+import os
+import sys
+import shutil
+import tempfile
+
+
+class StreamCapture(object):
```

Comment:
Why not use https://docs.pytest.org/en/latest/reference.html#capsys ?

Reply:
Because I didn't know about it :D